hack/jenkins/e2e-runner.sh

```diff
@@ -21,6 +21,11 @@ set -o nounset
 set -o pipefail
 set -o xtrace
 
+export HOME=${WORKSPACE} # Nothing should want Jenkins $HOME
```

Comment:
Since we have a `gcloud` invocation in e2e.sh, we may want to still set this there. (gcloud spits in `$HOME/.config` I think.)